installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -150,6 +150,8 @@ spec:
               virtualenv -p {{ custom_venv.python | default(custom_venvs_python) }} \
                 {{ custom_venvs_path }}/{{ custom_venv.name }} &&
               source {{ custom_venvs_path }}/{{ custom_venv.name }}/bin/activate &&
+              {{ custom_venvs_path }}/{{ custom_venv.name }}/bin/pip install {{ trusted_hosts }} -U psutil \
+                --upgrade pip  &&
```

Comment:
is the extra `--upgrade` necessary here (isn't that what `-U` does?)

Reply:
yes, apologies. -U is upgrade